dev-python/shellingham/shellingham-1.5.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="Tool to Detect Surrounding Shell"
+HOMEPAGE="
+	https://github.com/sarugaku/shellingham
+"
+SRC_URI="
+	https://github.com/sarugaku/shellingham/archive/${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
Add tests, please.

Reply:
In some gentoo documentation I've seen something about not having to include tests if it's not useful for testing on gentoo specifically, therefore I've not included them. I don't have the citation, maybe I misunderstood.

Reply:
Ah, so we avoid tests if they're purely something like `pylint` or coverage tests, but we want to run them otherwise.